release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3100,6 +3100,34 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-3"]
+=== RHSA-2023:7315 - {product-title} 4.14.3 bug fix and security update
+
+Issued: 2023-11-21
+
+{product-title} release 4.14.3, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7315[RHSA-2023:7315] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:7321[RHBA-2023:7321] advisory.
```

Comment:
```suggestion
{product-title} release 4.14.3, which includes security updates, is now available. The list of bug fixes included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7315[RHSA-2023:7315] advisory. The RPM packages included in the update can be found at link:https://access.redhat.com/errata/RHBA-2023:7321[RHBA-2023:7321].
```

Reply:
This is templated text as this stage that has went through a few rounds of approvals, so I'll leave the format as it. Thanks!